frontend/src/locales/de.json

```diff
@@ -608,6 +610,12 @@
         "success": "Erfolgreich registriert",
         "title": "Konto erstellen"
       },
+      "resendActivation": {
+        "errorMessage": "Neu senden der Aktivierung fehlgeschlagen.",
+        "send": "Abschicken",
+        "successMessage": "Mail mit Link für 'Aktivierung' wird versendet.",
```

Comment:
```suggestion
        "successMessage": "E-Mail mit Link für Aktivierung wird versendet.",
```

Reply:
Mail is more often than E-Mail in our current translations.
Will keep Mail.
(And it is consistent  to the resendPassword success message)